engine/openbor.h

```diff
@@ -1790,8 +1790,6 @@ if(n<1) n = 1;
 //#define     MAX_MOVES             16
 //#define     MAX_MOVE_STEPS        16
 
-#pragma pack(4)
```

Comment:
Have to be-careful with packing. If data structures are not written/read from disk or being transferred over communication protocols this flag can cause bad behaviors. I encountered an issue where the pointer being assigned with an array of 11 elements was fragment and yet the code is expecting contiguous memory storage.... mostly arm based devices would have trouble with this since x86 cpus will try to fetch the data twice from memory when not aligned.... not the case with arm.

Specifically related to memory alignment of the colorable:

typedef struct
{
    s_axis_plane_vertical_int graph_position;
    s_axis_plane_vertical_int name_position;
    s_axis_plane_vertical_int size;
    e_status_config config_flags;
    int barlayer;
    int backlayer;
    int borderlayer;
    int shadowlayer;
    int (*colourtable)[11]; //0 default backfill 1-10 foreground colours
} s_barstatus;